### DISCUSSION TITLE
Webview issue

### DISCUSSION THREAD

**Discussion by YogitaMagdum** [dotnet/maui#20308] [Q&A]
We are encountering an issue where the WebView in our Android app is automatically appending "file:///android_asset/" to the URL, causing the webpage not to display correctly on our device.
Please help us how to solve this issue.

**Comment by anila-mathew** [comment]
HI @YogitaMagdum Did u find any solution for this issue?? I am also facing the same issue now...

**Comment by MartyIX** [comment]
I can see `android_asset` here: 

https://github.com/dotnet/maui/blame/9b44c9a26d7dc2de3b75b36853cd689338e33e84/src/Core/src/Platform/Android/MauiWebView.cs#L9C3-L9C3

So it appears that `android_asset` is a standard asset folder for android apps.

However, there is this comment https://github.com/dotnet/maui/issues/12974#issuecomment-2088598771 which seems to explain how to access resources on Android:

> As a workaround you can load the .gif into a webview.
> 
>     1. Copy the .gif to `Resources/Raw/my.gif` folder of your project
> 
>     2. Add a .html file (see below) to `Resources/Raw/my.html`
> 
>     3. Add a web view: `<WebView HeightRequest="200" WidthRequest="200" x:Name="webView" />`
> 
>     4. In your code behind, set the source:
> 
> 
> ```cs
> webView.Source = new UrlWebViewSource {
>   Url = "file:///android_asset/my.html"
> };
> ```
> 
> HTML File eg:
> 
> ```
> <html>
> <body style="background-color: #ffffff; padding: 0; margin: 0;">
>     <img src="my.gif" width="200" height="200">
> </body>
> </html>
> ```
> 
> This works in release mode in my testing. It's not ideal, but it's a workaround until we can figure out and eventually fix why release mode is not animating.

**Comment by phamnakhoa** [comment]
HI @YogitaMagdum .I am have the same issue.Do you have a solution yet ?
